src/common/net/node-client.ts

```diff
@@ -40,6 +40,18 @@ export class NodeHttpClient extends HttpClient implements HttpClientInterface {
     return 'node';
   }
 
+  static override getBody(entity: unknown): string | null {
```

Comment:
I decided to override the parent class's `getBody` since it's return value of `BodyInit` (which is compatible with `fetch`) didn't make sense for the Node HTTP API.